cg/cli/backup.py

```diff
@@ -27,6 +34,35 @@ def backup(context: CGConfig):
     pass
 
 
+@backup.command("encrypt-flow-cells")
+@DRY_RUN
+@click.pass_obj
+def encrypt_flow_cells(context: CGConfig, dry_run: bool):
+    """Encrypt flow cells."""
+    status_db: Store = context.status_db
+    for flow_cell in get_flow_cells_from_path(flow_cells_dir=Path(context.flow_cells_dir)):
+        db_flow_cell: Optional[Flowcell] = status_db.get_flow_cell_by_name(
+            flow_cell_name=flow_cell.id
+        )
+        if db_flow_cell and db_flow_cell.has_backup:
+            LOG.debug(f"Flow cell: {flow_cell.id} is already backed-up")
+            continue
+        flow_cell_encryption_api = FlowCellEncryptionAPI(
+            binary_path=context.encryption.binary_path,
+            dry_run=dry_run,
+            encryption_dir=Path(context.backup.encrypt_dir),
+            flow_cell=flow_cell,
```

Comment:
I'd like it if this api was instantiated in the context instead, but I guess that is not possible if you want to create it per flow cell

Reply:
Great!